# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../apmd/skip_xapm.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2002-07-25 08:37:45.000000000 +0200
+++ ./Makefile	2002-07-25 08:39:39.000000000 +0200
@@ -28,7 +28,10 @@
 .SUFFIXES:
 
 OBJS=apmlib.o
-EXES=apm apmd xapm apmsleep
+EXES=apm apmd apmsleep
+ifndef SKIP_XAPM
+  EXES += xapm
+endif
 HEADERS=apm.h
 
 PREFIX=/usr
@@ -94,8 +97,10 @@
 	install        apmd            ${PREFIX}/sbin
 	install -b -Vt apmd_proxy      ${APMD_PROXY_DIR}
 	install -m 644 apmd.8          ${MANDIR}/man8
+ifndef SKIP_XAPM
 	install        xapm            ${PREFIX}/bin
 	install -m 644 xapm.1          ${MANDIR}/man1/xapm.1
+endif
 	install        apmsleep	       ${PREFIX}/bin
 	install -m 644 apmsleep.1      ${MANDIR}/man1
 	install -m 644 apmsleep.1.fr   ${MANDIR}/fr/man1
